pkg/scheduler/core/generic_scheduler.go

```diff
@@ -677,6 +677,9 @@ func PrioritizeNodes(
 	nodes []*v1.Node,
 	extenders []algorithm.SchedulerExtender,
 ) (schedulerapi.HostPriorityList, error) {
+	trace := utiltrace.New(fmt.Sprintf("Ranking nodes for %s/%s", pod.Namespace, pod.Name))
+	defer trace.LogIfLong(100 * time.Millisecond)
```

Comment:
The traces in #78208 show that PrioritizeNodes takes less than 100ms, so those traces may not show up at all.